File: 75-sort-colors/sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        zero = nums.count(0)
        one = nums.count(1)
        two = nums.count(2)
        for i in range(len(nums)):
            if zero:
                nums[i] = 0
                zero-=1
            elif one:
                nums[i] = 1
                one-=1
            else:
                nums[i] = 2
                two-=1
        print(zero)
        

File: 75-sort-colors/sort-colors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int zero = count(nums.begin(),nums.end(),0);
        int one = count(nums.begin(),nums.end(),1);
        int two = count(nums.begin(),nums.end(),2);
        cout<<zero<<" "<<one<<" "<<two<<"\n";
        for(int i = 0 ;  i < nums.size();i++){
            if(zero)nums[i] = 0,zero--;
            else if(one)nums[i]=1,one--;
            else nums[i] = 2,two--;
        }

    }
};